tests/utilities_02.cc

```diff
@@ -0,0 +1,34 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2014 by the deal.II authors
+//
+//    This file is subject to LGPL and may not be distributed
+//    without copyright and license information. Please refer
+//    to the file deal.II/doc/license.html for the  text  and
+//    further information on this license.
+//
+//-----------------------------------------------------------
+
+// Test if the "print" utilities function works as expected. This is
+// somewhat the inverse of the dealii::Utilities::split_string_list(),
+// but it works with arbitrary objects that support operator<<();
+
+#include "tests.h"
+#include "utilities.h"
+#include <deal.II/base/utilities.h>
+#include <deal.II/base/mpi.h>
+
+int main (int argc, char *argv[])
+{
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, numbers::invalid_unsigned_int);
```

Comment:
Guard this with 

~~~C++
#ifdef DEAL_II_WITH_MPI
Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, numbers::invalid_unsigned_int);
#endif 
~~~